### ISSUE TITLE
email should be case insensitive

### ISSUE THREAD

**Issue by PetrSnobelt** [anvilresearch/connect#344]
Hello,
I figured out that the email used for logging depends on uppercase and lowercase letters.

Is there any way how to disable this. I think email should be case insensitive.

Thank you 
Petr

**Comment by dmitrizagidulin** [comment]
I agree. Given that the underlying persistence (Redis) treats keys as case-sensitive, but emails are generally understood (by users, and by the actual email spec) to be case-insensitive, this gets us into several confusing situations.

**Comment by dmitrizagidulin** [comment]
Specifically, I think email addresses should be forced to lowercase by the registration code, at user signup / account creation.
